packages/puppeteer-core/Herebyfile.mjs

```diff
@@ -38,8 +38,7 @@ export const generateInjectedTask = task({
       entryPoints: ['src/injected/injected.ts'],
       bundle: true,
       format: 'cjs',
-      platform: 'browser',
-      target: 'ES2022',
+      target: 'chrome117',
```

Comment:
Let's stick with the previous part as we support Firefox,
I can see someone bumping chrome to new version and that version supporting ESNEXT that Firefox does not and breaking.